c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -378,6 +383,35 @@ typedef enum {
 #define LSM_SYSTEM_CACHE_PCT_NO_SUPPORT             -2
 #define LSM_SYSTEM_CACHE_PCT_UNKNOWN                -1
 
+typedef enum {
+    LSM_BATTERY_TYPE_UNKNOWN = 1,
+    LSM_BATTERY_TYPE_OTHER = 2,
+    LSM_BATTERY_TYPE_CHEMICAL = 3,
+    LSM_BATTERY_TYPE_CAPACITOR = 4,
+} lsm_battery_type;
```

Comment:
@BlaineEXE Could you point me the document you are referring? From the code I found, `lsm_pool_member_type` and `lsm_disk_type` seems said otherwise.